omero/sysadmins/repository-move.txt

```diff
@@ -0,0 +1,94 @@
+
+     # | Id | UUID                                 | Type    | Path
+    ---+----+--------------------------------------+---------+-----------------------------------------
+     0 | 1  | 309ea513-a23c-48d1-abd2-9ceed1b3ffa4 | Managed | /Users/omero/var/omero/ManagedRepository
+     1 | 2  | ScriptRepo                           | Script  | /Users/omero/dist/lib/scripts
+     2 | 3  | 3ec8c878-c776-48a3-b57e-2a11b0c97045 | Public  | /Users/omero/var/omero
+    (3 rows)
+
+If the Managed Repository is within the OMERO data directory and the whole
+data directory is to be moved then the following steps should be used:
+
+::
+
+    bin/omero config omero.data.dir NEW
+    mv OLD NEW
+
+For example, moving the OMERO data directory from ``/Users/omero/var/omero``
+to ``/Volumes/omero``:
+
+::
+
+    $ bin/omero admin stop
+    ...
+    $ bin/omero config omero.data.dir /Volumes/omero
+    $ mv /Users/omero/var/omero /Volumes/omero
+    $ bin/omero admin start
+    ...
+    $ bin/omero fs repos
+
+     # | Id | UUID                                 | Type    | Path
+    ---+----+--------------------------------------+---------+---------------------------------
+     0 | 1  | 309ea513-a23c-48d1-abd2-9ceed1b3ffa4 | Managed | /Volumes/omero/ManagedRepository
+     1 | 2  | ScriptRepo                           | Script  | /Users/omero/dist/lib/scripts
+     2 | 3  | 3ec8c878-c776-48a3-b57e-2a11b0c97045 | Public  | /Volumes/omero
+    (3 rows)
+
+If the Managed Repository is in a separate location to OMERO data directory
+or only the Managed Repository is to be moved then the following steps
+should be used:
+
+::
+
+    bin/omero config omero.managed.dir NEW
+    mv OLD NEW
+
+For example, moving the Managed Repository from ``/Users/omero/var/omero/ManagedRepository``
+to ``/Volumes/imports/ManagedRepository``:
+
+::
+
+    $ bin/omero admin stop
+    ...
+    $ bin/omero config omero.managed.dir /Volumes/imports/ManagedRepository
+    $ mv /Users/omero/var/omero/ManagedRepository /Volumes/imports/ManagedRepository
+    $ bin/omero admin start
+    ...
+    $ bin/omero fs repos
+
+     # | Id | UUID                                 | Type    | Path
+    ---+----+--------------------------------------+---------+-----------------------------------
+     0 | 1  | 309ea513-a23c-48d1-abd2-9ceed1b3ffa4 | Managed | /Volumes/imports/ManagedRepository
+     1 | 2  | ScriptRepo                           | Script  | /Users/omero/dist/lib/scripts
+     2 | 3  | 3ec8c878-c776-48a3-b57e-2a11b0c97045 | Public  | /Users/omero/var/omero
+    (3 rows)
+
+.. note::
+    If ``omero.managed.dir`` is not set then the location of the Managed
+    Repository will be determined by ``omero.data.dir`` and the OMERO
+    directory should only be moved as a whole.
+
+    If the Managed Repository needs to be moved to a location other than that
+    set by ``omero.data.dir``, to a location outside of the OMERO, for
+    example, data directory, then ``omero.managed.dir`` must be set.
```

Comment:
the "for example" in here breaks the flow awkwardly, though I don't off-hand have better ideas